CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ collection of all data, respectively. To learn more what data is collected in En
 read the notes of the `Enso 2.0.0-alpha.1` release.
 
 
+<br/>![New Features](/docs/assets/tags/new_features.svg)
+
+#### Visual Environment
+- [Added the ability to reposition visualisations.][1096] There is no an icon in the visualisation action
+  bar that allows dragging the visualisation. Once it has been moved there appears another icon that
+  will reset the position to the original position.
```

Comment:
1. There is a typo "no" -> "now"
2. There are 3 other English errors catched by Gramarly. Please, create a free Gramarly account and check your spelling there before changing CHANGELOG. Btw, I started recently doing it for my code comments as well - this automatic tool is amazing (I was using it for years when writing mails already)

Reply:
`LanguageTool` is better (multi-language support) and seems to even have an [IDE integration](https://plugins.jetbrains.com/plugin/12175-grazie)! So, I'm using that now. Much better than to having to paste back and forth.